<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>33</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Root Frames</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="rootFrames_lineEdit"/>
   </item>
   <item>
    <widget class="QToolButton" name="previous_toolButton">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="add_toolButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="remove_toolButton">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="next_toolButton">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="auto_pushButton">
     <property name="text">
      <string>Set Auto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="selectNode_pushButton">
     <property name="text">
      <string>Select Node</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
